partEZ/app/Http/Controllers/EventController.php

```diff
@@ -254,23 +252,10 @@ public function validatePoll( $eid )
 
     }
 
-    public function getVotes($pid, $oid)
-    {//TODO: what is this counting? Not the votes, but the options?
```

Comment:
Do we not still need this to call the new function in polloption.php? Or do we just make a direct call to the poll option instead of using the controller now?